### ISSUE TITLE
Renewal: append() takes exactly one argument (2 given)

### ISSUE THREAD

**Issue by sebix** [letsencrypt/letsencrypt#2822]
Using letsencrypt 0.5.0, I got this error when renewing my certificates (subcommand renew):
```shell
2016-04-13 07:29:24,428:INFO:letsencrypt.renewal:Cert not yet due for renewal
2016-04-13 07:29:24,429:DEBUG:letsencrypt.main:Exiting abnormally:
Traceback (most recent call last):
  File "/root/.local/share/letsencrypt/bin/letsencrypt", line 11, in <module>
    sys.exit(main())
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/main.py", line 692, in main
    return config.func(config, plugins)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/main.py", line 538, in renew
    renewal.renew_all_lineages(config)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/renewal.py", line 355, in renew_all_lineages
    renew_skipped, parse_failures)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/renewal.py", line 290, in _renew_describe_results
    notify(parse_failures, "parsefail")
TypeError: append() takes exactly one argument (2 given)
```

**Comment by kevinburke** [comment]
Happy to write a test here but I'm not seeing any tests for `renew_all_lineages` or `renew_describe_results` or Python tests that mention `renew`, maybe I'm not looking in the right place?

**Comment by bmw** [comment]
Fixed in #2860.
